function/values.yaml

```diff
@@ -0,0 +1,16 @@
+replicas: 0
```

Comment:
don't you want 1 instance? and then something like keda scales it?

Reply:
Ideally, Keda should scale from zero.

Reply:
are we doing keda from day one?

Reply:
may be not, but i like a function to have 0 replicas :)

Reply:
me too